locales/fr.js

```diff
@@ -237,9 +237,10 @@
 	"logging_in": "Connexion",
 	"password_confirmation": "Confirmation du mot de passe",
 	"confirm_again": "Veuillez consulter votre boîte pour confirmer votre email.",
-	"confirmed": "Adresse email confirmée avec succès. merci!",
+	"confirmed": "Adresse email confirmée avec succès. Merci !",
+	"signing_up": "Signing Up",
 	"password_check_inbox": "password_check_inbox",
-	"what_is_your_name": "Bonjour! Choisir un nom d'utilisateur s'il vous plaît.",
+	"what_is_your_name": "Bonjour ! Choisir un nom d'utilisateur s'il vous plaît.",
```

Comment:
Can you explain why you put spaces before exclamation marks?

Reply:
It's the French convention for punctuation.